src/app/pages/storage/snapshots/snapshot-list/snapshot-list.component.ts

```diff
@@ -273,8 +279,52 @@ export class SnapshotListComponent {
     });
   }
 
+  restructureData(selected: SnapshotData[]): DialogData {
+    const datasets: string[] = [];
+    const snapshots: any = {};
+    console.log(selected);
+    selected.forEach((item: SnapshotData) => {
+      if (!snapshots[item.dataset]) {
+        datasets.push(item.dataset);
+        snapshots[item.dataset] = [];
+      }
+
+      snapshots[item.dataset].push(item.snapshot);
+    });
+
+    return { datasets, snapshots };
+  }
+
+  getMultiDeleteMessage(selected: SnapshotData[]): string {
+    const grandTotal: string = selected.length > 1 ? selected.length.toString() : '';
+    let message = `<strong>The following ${grandTotal} snapshots will be deleted. Are you sure you want to proceed?</strong>`;
```

Comment:
This string needs to be translated with `this.translate.instant()` with grandTotal being a placeholder with {grandTotal}.

Reply:
Translations added.